internal/host/installcmd.go

```diff
@@ -54,7 +54,7 @@ func (i *installCmd) GetSteps(ctx context.Context, host *models.Host) ([]*models
 		role = models.HostRoleBootstrap
 	}
 
-	cmdArgsTmpl := "podman run -v /dev:/dev:rw -v /opt:/opt:rw {{if .HOST_CA_CERT_PATH}}-v {{.HOST_CA_CERT_PATH}}:{{.HOST_CA_CERT_PATH}}:rw {{end}}-v /run/systemd/journal/socket:/run/systemd/journal/socket --privileged --pid=host --net=host " +
+	cmdArgsTmpl := "podman run -v /dev:/dev:rw -v /opt:/opt:rw -v /etc/ssh:/etc/ssh {{if .HOST_CA_CERT_PATH}}-v {{.HOST_CA_CERT_PATH}}:{{.HOST_CA_CERT_PATH}}:rw {{end}}-v /run/systemd/journal/socket:/run/systemd/journal/socket --privileged --pid=host --net=host " +
```

Comment:
Instead of making the entire /etc/ssh writable, can we just bind mount the ssh public key?

Should /root/.ssh be used instead of /etc/ssh?

Reply:
Thanks for your review @rwsu .
About your first question, I can try (didn't know we can mount a single file).
About you second question, at this point of the code, `root/.ssh` doesn't exist, It is created in this PR https://github.com/openshift/assisted-installer/pull/80.

The default location of the public key is under `/etc/ssh`